python/paddle/fluid/transpiler/inference_transpiler.py

```diff
@@ -99,20 +148,20 @@ def fuse_batch_norm(self, program, place, scope):
             program (Program): program to transpile
             place (Place): inference place
             scope (Scope): inference Scope
-        
+
         '''
         self.scope = scope
         self.place = place
         self.block = program.block(0)
-        self.input_map = {}  # store the input names should be adjusted 
+        self.input_map = {}  # store the input names should be adjusted
 
         i = 0
-        while i < len(self.block.ops):
+        while i < len(self.block.ops) - 2:
```

Comment:
why do you change line 159?

Reply:
Because in lines 174 and 183 we access `i+2` element.